extensions/data-workspace/src/common/constants.ts

```diff
@@ -51,6 +51,8 @@ export const NameCannotBeEmpty = localize('dataworkspace.nameCannotBeEmpty', "Na
 export const TargetPlatform = localize('dataworkspace.targetPlatform', "Target Platform");
 export const SdkStyleProject = localize('dataworkspace.sdkStyleProject', "SDK-style project (Preview)");
 export const LearnMore = localize('dataworkspace.learnMore', "Learn More");
+export const Yes = localize('dataworkspace.yes', "Yes");
```

Comment:
Consider making this `Yes (Recommended)` - that's typically what we do when we want to push users to a specific type